__tests__/crawler.test.js

```diff
@@ -31,6 +31,21 @@ test("Test links to check if they're all reachable", async () => {
     /^https:\/\/www\.linkedin\.com\/company\/tigera\/?$/,
     'http://etcd.co',
     'https://success.docker.com/article/docker-ee-best-practices',
+    `http://heartbleed.com`,
+    `https://installer.calicocloud.io`,
+    `https://installer.calicocloud.io/charts`,
+    `https://docs.tigera.io/calico/charts`,
+    `https://sysctl-explorer.net/net/ipv4/fib_multipath_hash_policy`,
+    `https://github.com/projectcalico/calico/releases/download/master/ocp.tgz`,
+    `https://docs.tigera.io/master/reference/installation/api#operator.tigera.io%2fv1`,
+    `https://downloads.tigera.io/ee/master/download/binaries/master/calicoctl`,
+    `https://projectcalico.docs.tigera.io/manifests/calico-vxlan.yaml`,
+    ///^https?:\/\/localhost[\w\-.~:/?#[\]@!$&'()*+,;=]*$/,
+    /^https?:\/\/csrc\.nist\.gov\/projects\/cryptographic-module-validation-program\/certificate\/\d+$/,
+    /^https:\/\/installer\.calicocloud\.io:[0-9]{3,4}$/,
+    /^https:\/\/github\.com\/[\w-]+\/[\w-]+\/releases\/download\/[\w-]+\/install-calico-windows\.ps1$/,
```

Comment:
Should some of these be fixed in the source docs? We should always try to fix in the source docs before resorting to the skip list. Please review each of these with @ctauchen  - thank you.

Reply:
Sure Doug.

Reply:
Yeah, I'd say a few of those just need fixing at source:
* http://heartbleed.com has been dead quite a while now. Can we find an alternative or remove it?
* https://sysctl-explorer.net/net/ipv4/fib_multipath_hash_policy/ (It just needs the trailing slash! `/`)
* https://projectcalico.docs.tigera.io/manifests/calico-vxlan.yaml --> That definitely needs changing to a new URL with {{manifestsUrl}}. But I'm having trouble finding where this is appearing in the docs. Have you seen it?
* https://docs.tigera.io/master/reference/installation/api#operator.tigera.io%2fv1 Can definitely be changed.
* The .ps1 URL seems to be for all versions? I would expect to only be for /master/ for OS. 

One category of broken links is a number of 'master' resources, which are known not to be present. There's more involved in sorting these out. Last I heard, there was no immediate plan to maintain these. The idea is that we don't want people actually building stuff from /master/?

@girishshankaran I'll write a ticket to deal with these /master/ resources separately. For now, let's have them in the skip list.

Reply:
- http://heartbleed.cm: Found an [alternative link](https://owasp.org/www-community/vulnerabilities/Heartbleed_Bug).
- https://sysctl-explorer.net/net/ipv4/fib_multipath_hash_policy/: The trailing slash is there in the source. Check with @doucol why it is reported as dead. Does the crawler ignore trailing slashes in a URL?
- https://docs.tigera.io/master/reference/installation/api#operator.tigera.io%2fv1: Changed in the source
- The specific .ps1 URL is added to the skip list
- https://projectcalico.docs.tigera.io/manifests/calico-vxlan.yaml: Changed in the source. @ctauchen Please verify the fix.

Reply:
Getting close! My results from the test:
<img width="754" alt="image" src="https://user-images.githubusercontent.com/39195715/224333418-6bc21e46-18d0-44cf-8097-fc93414355bf.png">

Reply:
Added these links in the skip list:
```
http://localhost:4242/calico/3.24/network-policy/istio/files/30-policy.yaml
https://downloads.tigera.io/ee/master/download/binaries/master/calicoq
https://en.wikipedia.org/wiki/Autonomous_System_(Internet)%3E
 https://installer.calicocloud.io/manifests/v3.15.1-8/manifests
```

Reply:
I am curious about this one:
http://localhost:4242/calico/3.24/network-policy/istio/files/30-policy.yaml

Why is this not something we are fixing in the source docs?

Reply:
hint:  if you do a search for "(files/30-policy.yaml" - you will find the offending reference. It is, in fact, a bug in our source docs.

Reply:
@girishshankaran can you take a look at this?

Reply:
Yes Chris. I fixed the issue this morning. The issue seems to be with a missing slash (/) in the file path. @doucol Please correct me if I am wrong. I removed the link from the skip list as well. We need to run a test to make sure that the issue is resolved.

Reply:
> Yes Chris. I fixed the issue this morning. The issue seems to be with a missing slash (/) in the file path. @doucol Please correct me if I am wrong. I removed the link from the skip list as well. We need to run a test to make sure that the issue is resolved.

That is correct. It was missing a leading forward slash to make the path absolute, not relative to the mdx file.